mindsdb_sdk/utils/mind.py

```diff
@@ -16,27 +18,48 @@ def __init__(self, name):
         self.name = name
 
 
+class DataSourceConfig(BaseModel):
+    """
+    Represents a data source that can be made available to a Mind.
+    """
+    id: str = Field(default_factory=lambda: uuid4().hex)
+
+    # Description for underlying agent to know, based on context, whether to access this data source.
+    description: str
+
+
+class DatabaseConfig(DataSourceConfig):
+    """
+    Represents a database that can be made available to a Mind.
+    """
+
+    # Integration name (e.g. postgres)
+    type: str
+
+    # Args for connecting to database.
+    connection_args: dict
+
+    # Tables to make available to the Mind (defaults to ALL).
+    tables: List[str] = []
+
+
 # Create mind entity util function
 def create_mind(
         base_url: str,
         api_key: str,
         name: str,
-        description: str,
-        data_source_type: str,
-        data_source_connection_args: dict,
+        data_source_configs: List[DataSourceConfig] = None,
```

Comment:
An example in examples/using_database_mind_text2sql.py#L25 need to be updated (it uses this function)